### ISSUE TITLE
For more details about the learning rate

### ISSUE THREAD

**Issue by VClay** [amaralibey/Bag-of-Queries#10]
Thank you very much for your great work. Could you share the complete training code, especially the scheduling of the learning rate? For more details about the learning rate, I would like to know if it is immediately multiplied by 0.1 after the initial ten epochs warm up, or if it slowly decreases to the original 0.1 every ten epochs thereafter. My learning rate has changed as follows, but my training BoQ learning rate has changed as shown in the following figure, but it does not achieve better results. Can you provide some help? I would be very grateful.

![lr](https://github.com/user-attachments/assets/186af444-59b5-471e-b9e0-7c14e7627195)

**Comment by VClay** [comment]
This is another situation for learning rate scheduling. Which of these two situations is more suitable for your initial setup?
![lr_2](https://github.com/user-attachments/assets/960d39da-68e3-4afc-936f-de0b2aaeac50)

**Comment by VClay** [comment]
Thank you for your great reply. Is the learning rate of the backbone also 2e-4? The framework you provided is excellent!

**Comment by amaralibey** [comment]
Hey @VClay, I am glad that you like the OpenVPRLab framework.

You can see in lines 85-88 (https://github.com/amaralibey/OpenVPRLab/blob/main/src/core/vpr_framework.py) that we are actually using the same learning rate for the backbone and the BoQ. But, I have also tried with `0.2*self.lr` and `0.1*self.lr`. They all achieved > 93% R@1 on MSLS-val.

**Comment by VClay** [comment]
thanks
